tests/unistd/links.c

```diff
@@ -85,5 +84,10 @@ int main() {
 #endif
   errno = 0;
 
+  int result = access("/linkX", F_OK);
+  assert(result == -1);
+  assert(errno == 32);
```

Comment:
Can you use a EXX constant here.. its not clear what 32.

Also perhaps add a command either here or above to make it clear what we are testing?

Reply:
How's it look now?